tests/ProxyManagerTest/Functional/LazyLoadingGhostFunctionalTest.php

```diff
@@ -1353,4 +1387,14 @@ function (VoidCounter $proxy, $method, $params, & $initializer, array $props) us
 
         self::assertSame($initialCounter + $increment, $proxy->counter);
     }
+
+    private static function isPropertyInitialized(object $object, ReflectionProperty $property) : bool
+    {
+        return array_key_exists(
```

Comment:
You should be able to use `$property->isInitialized($object)` here.

Reply:
Does that also consider the `unset()` scenario?

Reply:
Yes.

Reply:
Hm actually, is this one called on an object with lazy initialization __get? Because I think isInitialized() is going to invoke that.